text/0000-lifecycle-experimental-api-mode.md

```diff
@@ -0,0 +1,139 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle Experimantal API Mode and Prelease APIs
+- Start Date: 2020-08-25
+- Author(s): @hone
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the addition of a third API mode `experimental` and extends [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md).
+
+# Definitions
+[definitions]: #definitions
+
+experimental - a new API mode that is marked as experimental. Unlike supported and deprecated APIs, this may be for an unreleased API. The purpose is to gather feedback that will be used to improve the design before it's finalized. It will not carry the same promise of stability or compatibility.
+
+# Motivation
+[motivation]: #motivation
+
+In order to have a smooth transition to 1.0 of the spec, there needs to be a way for the project to test out these ideas in the wild and gather feedback. Even beyond 1.0, this desire still stands for testing out concepts in an organized fashion where the feedback can be provided. This feedback will help us improve the design before we finalize it. The quality of feedback we get from users about their real life usage of experimental APIs will help us avoid problems we may miss when approving RFCs.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This proposes two changes to Lifecycle:
+* Prerelease API versions which is used for testing API / spec changes.
+* `experimental` is a new API mode for testing implementation of the API similar to the modes defined in [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md).
+
+## Prerelease API
+
+A prerelease API is a non-finalized API but is in a testable state. The the API version must be of the format: `<major>.<minor>-<alphanumeric>`, i.e. `1.0-alpha1`. Though these aren't final, there will be a tagged and release version of the appropriate spec for it. This way users can see what is supported when it's used. In most cases, these will be cut from the corresponding branch in `buildpacks/spec`, i.e. Buildpack API `0.5-alpha1` would be cut from the `buildpack/0.5` branch.
+
+When Lifecycle supports a prerelease API, it can be treated like any other API version and be used in any mode: `experimental`, `supported`, `deprecated`.
+
+## Experimental API Mode
```

Comment:
From the WG meeting today, @ekcasey was pushing to have this removed until it's needed.